e2e/gui/src/test/java/com/epam/pipeline/autotests/DavMountStorageLimitTest.java

```diff
@@ -0,0 +1,339 @@
+    private final String storage1 = format("davMountLimitStorage%s", Utils.randomSuffix());
+    private final String storage2 = format("davMountLimitStorage%s", Utils.randomSuffix());
+    private final String storage3 = format("davMountLimitStorage%s", Utils.randomSuffix());
+    private final String endpoint = format("%s%s", C.WEBDAV_ADDRESS, user.login.toUpperCase());
+    private final String errorMessage = "Max value for dav mounted storages is reached!" +
+            " Can't request this storage to be mounted, increase quotas!";
+    private final String requestInfo = "Description of the Request Filesystem access feature";
+    private final String doneRequestInfo = "Help tips - how to use the Filesystem access";
+    private final String uiPipeFileBrowserJson = readResourceFully(UI_PIPE_FILE_BROWSER_JSON);
+    private final int durationSeconds = 60;
+    private final String userRoleGroup = C.ROLE_USER;
+
+    @BeforeClass
+    public void createStorages() {
+        Stream.of(storage1, storage2, storage3)
+                .forEach(stor -> library()
+                    .createStorage(stor));
+    }
+
+    @BeforeMethod
+    void openApplication() {
+        logoutIfNeeded();
+        loginAs(admin);
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void removeStorages() {
+        Utils.removeStorages(this, storage1, storage2, storage3);
+    }
+
+    @Test
+    @TestCase(value = {"2233_1"})
+    public void checkWebdavAccessDurationPreference() {
+        final String[] webdavAccessDurationInitial = navigationMenu()
+                .settings()
+                .switchToPreferences()
+                .getLinePreference(storageWebdavAccessDuration);
+        try {
+            navigationMenu()
+                    .settings()
+                    .switchToPreferences()
+                    .setPreference(storageWebdavAccessDuration, String.valueOf(durationSeconds), true)
+                    .saveIfNeeded();
+            library()
+                .selectStorage(storage1)
+                .clickEditStorageButton()
+                .clickOnPermissionsTab()
+                .addNewGroup(userRoleGroup)
+                .selectByName(userRoleGroup)
+                .showPermissions()
+                .set(READ,ALLOW)
+                .set(WRITE,ALLOW)
+                .closeAll();
+            logout();
+            loginAs(user);
+            library()
+                .selectStorage(storage1)
+                .showMetadata()
+                .ensure(FILE_SYSTEM_ACCESS, enabled)
+                .click(FILE_SYSTEM_ACCESS)
+                .ensure(FILE_SYSTEM_ACCESS, text(accessEnabledMessage()))
+                .ensure(DISABLE, enabled)
+                .sleep(10, SECONDS)
+                .inAnotherTab(webdavpage ->
+                        checkWebDavPage(() -> new ToolPageAO(endpoint)
+                                .assertPageTitleIs(format("Index of /webdav/%s", user.login.toUpperCase()))
```

Comment:
"Index of /webdav/%s" can be introduced a constant

Reply:
Done